.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+  workflow_dispatch:
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Zip
+        uses: montudor/action-zip@v1
+        with:
+          args: zip --recurse-patterns godot-editor-debugger.zip "addons/**/*"
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          files: godot-editor-debugger.zip
```

Comment:
It only makes them on tagged pushes. So if you use 
```sh
git tag v2.1.0
```
then
```sh
git push && git push --tags
```
it will create a release; otherwise, it won't. It will also only create a release if the tag is in the form `v*.*.*`. Any other tag will be ignored.

Reply:
Also I actually dont even have version branches in this plugin yet, as I'm not working on it often (Even in other repos I dont even use `git tag` but rather make a branch directly, because then I dont have to use any different git commands xD). I'm just not sure I want this for now

Reply:
That's fair, I'll remove it